docs/ref/django-admin.txt

```diff
@@ -410,6 +412,12 @@ On PostgreSQL, models are also created for foreign tables.
 
 Specifies the database to introspect. Defaults to ``default``.
 
+.. django-admin-option:: --include-partitions
+
+.. versionadded:: 2.2
+
+If this option is provided, models are also created for partitions.
```

Comment:
This is PostgreSQL-specific then? I'm a bit annoyed at the prospect of having to modify the command for database-specific things but maybe there's no better option. I think about what MS SQL would do if this situation applied to some feature there.

Reply:
No, it isn't. Would we like to extend [#29722](https://code.djangoproject.com/ticket/29722) for adding support for other backends? Obviously this depends on whether the backend supports the feature and whether there is any way to introspect it. It looks like `INFORMATION_SCHEMA.PARTITIONS` is available for [MySQL](https://dev.mysql.com/doc/refman/8.0/en/partitions-table.html) and there are other options for [Oracle](https://docs.oracle.com/cd/E18283_01/server.112/e16541/part_admin005.htm).

| | PostgreSQL | MySQL | Oracle | SQLite |
| :--- | :---: | :---: | :---: | :---: |
| Partitions | ✔ | ✔ | ✔ | ✘ |
| Foreign Tables | ✔ | ✘ | ✘ | ✘ |
| Materialized Views | ✔ | ✘ | ✔ | ✘ |

For introspection of materialized views, I think we could use `USER_MVIEWS` for [Oracle](https://docs.oracle.com/cd/B28359_01/server.111/b28320/statviews_5332.htm#REFRN26139).